apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/AdminServiceAddressLocator.java

```diff
@@ -60,7 +60,7 @@ public void init() {
     refreshServiceAddressService =
         Executors.newScheduledThreadPool(1, ApolloThreadFactory.create("ServiceLocator", false));
 
-    refreshServiceAddressService.schedule(new RefreshAdminServerAddressTask(), 1, TimeUnit.MILLISECONDS);
+    refreshServiceAddressService.schedule(new RefreshAdminServerAddressTask(), 0, TimeUnit.MILLISECONDS);
```

Comment:
there should be no difference changing the initial delay from 1ms to 0ms..

Reply:
you are right.